config/initializers/rollbar.rb

```diff
@@ -0,0 +1,53 @@
+Rollbar.configure do |config|
+  config.access_token = Rails.application.secrets.rollbar_token
+
+  # Without configuration, Rollbar is enabled in all environments.
+  # To disable in specific environments, set config.enabled=false.
+  config.enabled = Rails.env.production? && config.access_token.present?
+
+  # By default, Rollbar will try to call the `current_user` controller method
+  # to fetch the logged-in user object, and then call that object's `id`,
+  # `username`, and `email` methods to fetch those properties. To customize:
+  config.person_method = nil
```

Comment:
I would wonder if this would be useful for debugging, e.g., does the issue affect all users or one specific user. Would it be worth obfuscating the user's identity by only exposing the id attribute, rather than turning the feature off? E.g.,

config.person_id_method = "id"
config.person_username_method = "id"
config.person_email_method = "id"